upstream-community-operators/lightbend-console-operator/lightbend-console-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,261 @@
+          "elasticsearchMemoryRequest": "510Mi",
+          "elasticsearchVersion": "7.2.0",
+          "enableElasticsearch": false,
+          "esConsoleExposePort": 30080,
+          "esConsoleImage": "{{.Values.imageCredentials.registry}}/enterprise-suite/es-console",
+          "esConsoleVersion": "v1.2.6",
+          "esGrafanaEnvVars": null,
+          "esGrafanaImage": "{{.Values.imageCredentials.registry}}/enterprise-suite/es-grafana",
+          "esGrafanaVersion": "v0.3.0",
+          "esGrafanaVolumeSize": "32Gi",
+          "esMonitorImage": "{{.Values.imageCredentials.registry}}/enterprise-suite/console-api",
+          "esMonitorVersion": "v1.2.3",
+          "exposeServices": false,
+          "goDnsmasqImage": "lightbend-docker-registry.bintray.io/lightbend/go-dnsmasq",
+          "goDnsmasqVersion": "v0.1.7-1",
+          "imageCredentials": {
+            "registry": "lightbend-docker-commercial-registry.bintray.io",
+            "username": "setme",
+            "password": "setme"
+          },
+          "imagePullPolicy": "IfNotPresent",
+          "kubeStateMetricsImage": "gcr.io/google_containers/kube-state-metrics",
+          "kubeStateMetricsVersion": "v1.2.0",
+          "minikube": false,
+          "podUID": null,
+          "prometheusDomain": "prometheus.io",
+          "prometheusImage": "prom/prometheus",
+          "prometheusMemoryRequest": "250Mi",
+          "prometheusVersion": "v2.9.2",
+          "prometheusVolumeSize": "256Gi",
+          "rbacApiVersion": "rbac.authorization.k8s.io/v1",
+          "usePersistentVolumes": true
+        }
+      }]
+    categories: Monitoring
+    certified: 'false'
+    createdAt: '2019-09-18T00:00:00Z'
+    description: >-
+      Lightbend Console provides visualizations for Akka, Play, and Lagom
+      applications. See https://developer.lightbend.com/docs/console/current/
+      for details.
+    containerImage: 'lightbend-docker-registry.bintray.io/lightbend/console-operator:1.2.3-rc2'
+    support: 'Lightbend, Inc.'
+    capabilities: Basic Install
+    repository: 'https://github.com/lightbend/console-charts/tree/master/operator'
+spec:
+  displayName: Lightbend Console Operator
+  description: >
+    ## About the managed application
+
+    Lightbend Console provides visualization and basic monitoring for Akka,
+    Play, and Lagom applications. See
+    https://developer.lightbend.com/docs/console/current/ for further details.
+
+    ## About this Operator
+
+    The operator provides a simple installation of Lightbend Console. See
+    https://github.com/lightbend/console-charts/blob/master/operator/README.md
+    for details.
+
+    ## Prerequisites for enabling this Operator
+
+    None are required to install the operator.
+
+    To install an instance of Console, you will need to add at minimum your
+    credentials to the Console custom resource.
+  maturity: alpha
+  version: 0.0.1
+  replaces: lightbend-console-operator.v0.0.1
```

Comment:
This should replace the older version, not the same version.  If not replacing the older version, then  remove the replace metadata